### PR TITLE
allow to load sibling calls

### PR DESCRIPTION
**Pull request by tmcgroul**

### CODE REVIEW

**Comment by belopash** [comment]
@tmcgroul is it "all extrinsic calls" or "calls with the same parent call"?

**Comment by tmcgroul** [comment]
@belopash calls with the same parent

**Comment by belopash** [comment]
@tmcgroul maybe it is better to introduce "extrinsicCalls"? This could potentially cover more cases

File: substrate/substrate-processor/src/mapping.ts [hunk 1]
```diff
@@ -219,6 +220,15 @@ export class Call {
         this.#subcalls = calls
     }
 
+    get siblings(): Call[] {
```
Comment (belopash) [inline substrate/substrate-processor/src/mapping.ts:223]: this new field looks a bit unnatural. You already can achieve the same by using `call.parent.subcalls`

**Comment by tmcgroul** [comment]
closing due to "some" open questions and revealing that the task isn't really necessary 🤷‍♂️

**Pull request closed without merging**